app/views/ChangeAvatarView/AvatarUrl.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import I18n from '../../i18n';
+import { FormTextInput } from '../../containers/TextInput';
+import { useDebounce, isImage, isValidURLRequest, isValidURL } from '../../lib/methods/helpers';
+
+const AvatarUrl = ({ submit }: { submit: (value: string) => void }) => {
+	const handleChangeText = useDebounce(async (value: string) => {
```

Comment:
There's no reason to debounce this.
You're doing it for URL and not for a search, so what's most likely to happen is to fail 99% of the time.
You added a `submit` prop for an `onChangeText` event.
It's awkward, because it changes the semantic.

What you should do is to validate on a form submit.